internal/agent/device/config/controller.go

```diff
@@ -33,10 +35,26 @@ func NewController(
 	}
 }
 
+func (c *Controller) init(ctx context.Context, current *v1alpha1.RenderedDeviceSpec) {
```

Comment:
I would expect any initialization to take place during bootstrap related to after reboot to take place in bootstrap.

Reply:
OK